articles/libraries/machine-learning/basic-classification.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Quantum machine learning library
+author: geduardo
+ms.author: v-edsanc@microsoft.com
+ms.date: 02/16/2020
+ms.topic: article
+uid: microsoft.quantum.libraries.machine-learning.basics
+---
+
+# Basic classification: classify wines with the QDK
+
+In this Quickstart, you will learn how to execute a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the QDK. It's okay if you don't understand everything, this is a quick overview of a complete QDK classifier. You will get more details explained as you go through the rest of the tutorials.
+
+In this guide we will use the [wine dataset](https://archive.ics.uci.edu/ml/datasets/wine) from the [UCI Machine Learning Repository](https://archive.ics.uci.edu/ml/datasets/wine), using a classifier structure defined in Q#. This dataset is included in the collection of standard datasets in the Quantum Machine Learning library. This means that we can load it directly from Q#.
+
+## Execute the classifier from the host program
+
+First, let's take a look at the host programs to see how the process looks like. It consists of three parts:
+
+- Initialization of the target machine, in this case a quantum simulator.
```

Comment:
This step is implicit rather than explicit in the Python host; not sure the best way to reflect that here.